### PR TITLE
test: skip test

### PR DESCRIPTION
**Pull request by vanpham-sw**
### 1. Why is this change necessary?

- To activate the major flag testing not only in our nightly but in all our runs.

### 2. What does this change do, exactly?

- Skip failing test because of the changes in 6.7, with tickets linked where possible

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-implement-new-changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfill them.

### CODE REVIEW

File: tests/acceptance/package.json [hunk 1]
```diff
@@ -27,9 +27,9 @@
   "dependencies": {
     "@currents/playwright": "^1.5.11",
     "@playwright/test": "1.49.1",
-    "@shopware-ag/acceptance-test-suite": "^10.2.3",
+    "@shopware-ag/acceptance-test-suite": "^11.1.0",
```
Comment (patzick) [inline tests/acceptance/package.json]: this should not be a part of this PR, dependencies bump should be a separate PR

File: tests/acceptance/tests/Accessibility/StorefrontAccessibility.spec.ts [hunk 2]
```diff
@@ -18,7 +18,15 @@ test('The Storefront should implement accessibility best practices.', { tag: '@A
...
     StorefrontAccountAddresses,
     StorefrontAccountPayment,
+    InstanceMeta,
 }) => {
+
+    // eslint-disable-next-line
+    if (InstanceMeta.features['V6_7_0_0']) {
+        // eslint-disable-next-line playwright/no-skipped-test
+        test.skip(true, 'This test is incompatible with V6_7_0_0');
+    }
```
Comment (patzick) [inline tests/acceptance/tests/Accessibility/StorefrontAccessibility.spec.ts]: linter rules exist to not be omitted :)

```suggestion
    test.skip(InstanceMeta.features['V6_7_0_0'], 'This test is incompatible with V6_7_0_0');
```

**Review by patzick — CHANGES_REQUESTED** [review]
Hey!
- pr description is missing for context, reviewer needs to find it from the code
- I'd not bypass linter rules, better to work with them (see code suggestion)
- dependencies bump (and lockfile changes) should be a separate PR to avoid conflicts

Conditional skip rule is added here: https://github.com/shopware/shopware/pull/5931/files#diff-e54f782f884d325fa4b4185de6674df3486ddc72e109791692b90f213ec91ba8R19 in #5931 so can be used after the merge

File: tests/acceptance/tests/Account/CustomerRegistration.spec.ts [hunk 3]
```diff
@@ -6,7 +6,15 @@ test('As a new customer, I must be able to register in the Storefront.', { tag:
...
     IdProvider,
     Register,
+    InstanceMeta,
 }) => {
+
+    // eslint-disable-next-line
+    if (InstanceMeta.features['V6_7_0_0']) {
+        // eslint-disable-next-line playwright/no-skipped-test
+        test.skip(true, 'This test is incompatible with V6_7_0_0');
+    }
```
Comment (frobel) [inline tests/acceptance/tests/Account/CustomerRegistration.spec.ts]: Not necessary to skip that test. It **was** a caching problem and it is solved now. (Please double-check with last nightly build)

**Review by patzick — APPROVED** [review]
Much cleaner 🙌

**Review by pweyck — APPROVED** [review]

**Pull request merged**